Handler/ContainerBasedHandlerLocator.php

```diff
@@ -26,6 +26,10 @@ class ContainerBasedHandlerLocator implements HandlerLocator
      */
     public function __construct(ContainerInterface $container, $commandToServiceIdMapping)
     {
+        if (empty($commandToServiceIdMapping)) {
```

Comment:
We can certainly remove this but it seemed to me like a handler locator for a bus that has no handler mappings is going to results in confusing behavior. This could happen if...

* The user has yet to add mappings (this would alert them... we could have a friendlier error here)
* The user has handlers tagged but not for one of the buses they created (multi-bus variant of previous issue and notifying via exception would still be good)
* The user set the default_bus config to a bus they didnt add to the commandbus config. This would result in an empty handler locator attempting to be created. We could validate that default_bus config value is a valid bus

I can't think of a situation where it should be allowed to create a locator that will never find any handlers.